tools/ci/ci_agent.py

```diff
@@ -88,6 +88,17 @@ def cli_run(args):
     _check_status(result)
 
 
+def cli_code_coverage(args):
```

Comment:
Any (one-line) comment(s) around functions?

Reply:
I will update all docs together

Reply:
We should do doc strings at PR time. They should describe the arguments also. Especially when passing a dict like this.

Reply:
Looks good.